native/more/add-friends-modal.react.js

```diff
@@ -0,0 +1,309 @@
+// @flow
+
+import React from 'react';
```

Comment:
Nit: I prefer to import everything `import * as React from 'react';`, open to being convinced otherwise though